webrender/src/device/gfx/device.rs

```diff
@@ -2471,6 +2465,9 @@ impl<B: hal::Backend> Device<B> {
         filter: TextureFilter,
     ) {
         debug_assert!(self.inside_frame);
+        if self.inside_render_pass || self.bound_draw_fbo == DEFAULT_DRAW_FBO {
```

Comment:
can we be rendering to a default FBO without an open render pass here? It looks like we should have only the first part as a condition, and the second would go into an `assert!` or something like that

Reply:
Thanks for noticing this, I think we don't need the second condition at all. The purpose of this PR is to have blits to non main targets as well, and blitting to `DEFAULT_DRAW_FBO` (main target) should always happen inside a render pass as well.